pkgs/development/python-modules/textx/default.nix

```
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, python3
+, fetchFromGitHub
+, mkdocs
+, twine
+, arpeggio
+, click
+, future
+, coverage
+, coveralls
+, flake8
+, html5lib
+, jinja2
+, pytest
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "textx";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-uZlO82dKtWQQR5+Q7dWk3+ZoUzAjDJ8qzC4UMLCtnBk=";
+  };
+
+  nativeBuildInputs = [
+    mkdocs
+    twine
+  ];
+
+  propagatedBuildInputs = [
+    arpeggio
+    click
+    future
+  ];
+
+  checkInputs = [
+    click
+    coverage
+    coveralls
+    flake8
+    html5lib
+    jinja2
+    pytest
+    tox
```

```suggestion
    html5lib
    jinja2
    pytestCheckHook
```
we don't want to lint the program because linter updates shouldn't break it and we want to execute the commands direct, not through tox.